hobby-blog.html

```diff
@@ -0,0 +1,71 @@
+        <h2>Crested Butte</h2>
+
+        <p>
+          Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
+        </p>
+
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque consectetur sed velit sit amet feugiat. Vestibulum quis mi lobortis, efficitur enim eu, auctor mauris. Integer lorem sapien, hendrerit id metus ut, dictum rhoncus est. Maecenas pharetra viverra urna, ultricies euismod metus feugiat sed. Vivamus id semper turpis. Praesent et dignissim diam. Sed nec sem sit amet risus ultricies pulvinar. Integer eu euismod urna, vitae luctus tortor. Duis suscipit ut sapien eget posuere. Aliquam finibus nisi eget velit fringilla ultrices eu finibus tellus. Vivamus mattis arcu quis rutrum ullamcorper. Phasellus lacus velit, auctor at lacinia non, tincidunt a risus. Praesent semper mauris id nulla ultrices, nec ultrices massa cursus.
+        </p>
+
+        <p>
+          Nunc nulla nisi, rhoncus non laoreet nec, finibus eget dolor. Integer in nunc in odio imperdiet elementum ut quis nunc. Morbi quam felis, lobortis ut dictum ut, sollicitudin nec nisl. Sed posuere placerat luctus. Morbi placerat hendrerit mattis. Praesent venenatis orci ac ante accumsan, hendrerit vulputate sem sodales. Etiam sollicitudin finibus ante, vehicula efficitur nibh porttitor eu. Aliquam varius ultrices dolor, a faucibus magna. Vivamus sodales accumsan turpis et ornare. Donec scelerisque faucibus lectus, at elementum eros maximus vitae. Integer justo risus, congue ac facilisis in, lacinia sit amet arcu. Sed tempus nisl nec tortor semper mollis. Duis non blandit arcu.
+        </p>
+
+      </section>
+
+      <section class = "gallery">
+        <div><img src="images/CB_401.jpg"/></div>
+        <div><img src="images/IMG_1369.jpg"/></div>
+        <div><img src="images/IMG_2689.jpg"/></div>
+        <div><img src="images/IMG_3143.jpg"/></div>
+        <div><img src="images/mtb_campfire.jpg"/></div>
+        <div><img src="images/mtb_carmtns.jpg"/></div>
+        <div><img src="images/mtb_headstand.jpg"/></div>
+        <div><img src="images/mtb_watercrossing.jpg"/></div>
+        <div><img src="images/mtb_wheel.jpg"/></div>
+      </section>
+
+      <section>
+        <img src="image/Moab_slickrock.jpg" />
```

Comment:
I found the image, but it looks like my computer care more about the case of the file extension. the file you saved has the extension ".JPG".